docs/DeveloperGuide.md

```diff
@@ -467,9 +467,13 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an empty folder.
+   
+   2. Open a command terminal, cd into the folder that you put the jar file in.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   3. Run the jar file with the command in the terminal java -jar ugteach.jar
```

Comment:
the command can be wrapped in the code too

Reply:
Thanks for the suggestion! Ive wrapped both of it.